lxd/instances_post.go

```diff
@@ -394,6 +402,10 @@ func createFromMigration(d *Daemon, r *http.Request, projectName string, req *ap
 }
 
 func createFromCopy(d *Daemon, r *http.Request, projectName string, req *api.InstancesPost) response.Response {
+	if d.cluster.LocalNodeIsEvacuated() {
```

Comment:
Don't we have a 4th one of those for `createFromMigration`?

Reply:
We cannot restrict `createFromMigration` as we need to function for the migration. If I add the restriction, the test fails.

Reply:
We can tell whether the request is coming from the cluster or not though.

`r.Context().Value(request.CtxProtocol)` will be set to `cluster` when dealing with an internal request.

Reply:
@stgraber oh that's new isn't it? I didn't know about that. I thought we still needed to inspect the custom HTTP headers to detect an internal request.

Reply:
Yeah, I think that was added when we did the plumbing for the requestor in the lifecycle events.